### PR TITLE
Bump main to Preview 7

### PR DESCRIPTION
**Pull request by carlossanlop**

### CODE REVIEW

**Review by jkoritzinsky — APPROVED** [review]

File: eng/Versions.props [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <PackageVersionNet7>7.0.5</PackageVersionNet7>
     <PackageVersionNet6>6.0.$([MSBuild]::Add($([System.Version]::Parse('$(PackageVersionNet7)').Build),11))</PackageVersionNet6>
     <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
-    <PreReleaseVersionIteration>6</PreReleaseVersionIteration>
+    <PreReleaseVersionIteration>7</PreReleaseVersionIteration>
```
Comment (carlossanlop) [inline eng/Versions.props:13]: @lewing @steveisok @radical the failure looks related.

I'm having a deja vu. Did we have a similar failure last month? If yes, should we modify the "main bumping process" to make sure the PR includes all the necessary changes that would avoid this failure?

Comment (radical) [reply eng/Versions.props:13]: The corresponding emsdk update was needed.

**Comment by radical** [comment]
Wasm debugger tests failure is unrelated.

**Comment by carlossanlop** [comment]
Thanks, @radical!

**Pull request merged**